content/attacks/posts/2011-10-05-Bitcoin7.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2011-10-05
+target-entities: Bitcoin7
+entity-types:
+  - Exchange
+  - Custodian
+attack-types: Infrastructure Attack
+title: Bitcoin7 Hacked, Losing 5000 BTC
+---
+
+## Summary
+
+On October 5, 2011, Bitcoin7, a cryptocurrency exchange, suffered a security breach that resulted in the loss of approximately 5,000 BTC. The attack was facilitated by a combination of weak security practices and system vulnerabilities, which allowed the attackers to gain unauthorized access to the exchange's hot wallet. Bitcoin7 claimed to have compensated affected customers and improved security measures following the attack, resuming normal operations within a few days.
+
+## Attackers
+
+The attackers behind the Bitcoin7 hack remain unidentified. Given the level of sophistication and knowledge required to exploit the system vulnerabilities and gain unauthorized access to the exchange's hot wallet, it is suspected that the attackers might have been a highly skilled group of hackers.
```

Comment:
Leave only the first sentence, and try to add on-chain information like attacker's Bitcoin address and etc. The section should contain only attacker's related factual information.